__init__.py

```diff
@@ -129,6 +129,29 @@ def currency_filter(num, grouping=True):
         return locale.currency(num, grouping=grouping)
 
 
+def _unique(values):
+    ret = []
+    for value in values:
+        if value not in ret:
+            ret.append(value)
+    return ret
+
+
+def _fmt_chart_tick(value):
+    return parse_date(value).strftime('%m/%y')
+
+
+@app.template_filter('fmt_chart_ticks')
```

Comment:
Nitpick, but we don't need to specify the filter name if it's the same as the method name... Though I could see one arguing for just always specifying it for consistency. What do you think?